syntax = "proto3";

import "tweetMessages.proto";

service RandomTime {
    rpc ReturnRandomTime (RandomTimeIntervalRequest) returns (RandomResponse);
}
